ui/imports/shared/views/chat/ImageContextMenu.qml

```diff
@@ -9,24 +9,25 @@ StatusMenu {
     property string imageSource
     property string domain
     property bool requireConfirmationOnOpen: false
+    property bool isGif: root.imageSource.toLowerCase().endsWith(".gif")
+    property bool isVideo: root.imageSource.toLowerCase().endsWith(".mp4")
```

Comment:
😭 could be `avi`, `mpg` or some random url that redirects to a video. We shouldn't rely on the file extension

Reply:
I agree with both points... but this is hard to do with remote URLs